java-file-formats/word-library/find-item-in-word-document.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Find item in Word document in Java | Syncfusion
+description: Find an item in the Word document in Java using Syncfusion Java Word library (Essential DocIO)
+platform: java-file-formats
+control: Word Library
+documentation: UG
+---
+
+# Find item in Word document
+
+Just like you can search for a text in a Word document, you can also search for an item (like image, content control, textbox, and so on). The .NET Word (DocIO) library supports to find an item in Word document based on its properties. With this functionality, you can:
+
+* Find the first item based on one property.
+* Find the first item based on multiple properties
+* Find all the items based on one property.
+* Find all the items based on multiple properties.
+
+## Find item by property
+
+Using FindItemByProperty API, find the first item in the Word document that has the specified property name and value.
```

Comment:
In Java API naming standard are different, so please use as per

Using findItemByProperty API,....

Reply:
Adressed